src/app/home/home.module.ts

```diff
@@ -1,15 +1,24 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { AngularFireModule } from '@angular/fire'
+import { AngularFirestoreModule } from '@angular/fire/firestore'
 
 import { HomeComponent } from './home.component';
 import { BannerComponent } from './banner/banner.component';
 import { EventsComponent } from './events/events.component';
 import { EventsService } from './events/events.service';
 import { SharedModule } from '../shared/shared.module';
+import { PartnersComponent } from './partners/partners.component';
+import { environment } from 'src/environments/environment';
 
 @NgModule({
-  declarations: [HomeComponent, BannerComponent, EventsComponent],
-  imports: [CommonModule, SharedModule],
+  declarations: [HomeComponent, BannerComponent, EventsComponent, PartnersComponent],
+  imports: [
+    CommonModule,
+    SharedModule,
+    AngularFireModule.initializeApp(environment.firebase),
+    AngularFirestoreModule
```

Comment:
Adding this here is not necessary, since these two AngularFire modules are already imported in the CoreModule